### PR TITLE
CLDR-16956 BRS v44 generate charts (which updates this file)

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-16956

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
```
Error:  (TestCLDRLocaleCoverage.java:416)  Error: coverageLocales.containsAll(localesForNames) - add to %language80 or lower under coverageLevels.xml?: Missing:	4
	blo	Anii
	kxv	Kuvi
	vec	Venetian
	xnr	Kangri
```

File: common/properties/coverageLevels.txt [hunk 1]
```diff
@@ -73,12 +75,17 @@ ko ;	modern ;	Korean
 kok ;	modern ;	Konkani
 ks ;	basic ;	Kashmiri
 ks_Deva ;	basic ;	Kashmiri (Devanagari)
+ku ;	moderate ;	Kurdish
+kxv ;	basic ;	Kuvi
+kxv_Deva ;	basic ;	Kuvi (Devanagari)
+kxv_Orya ;	basic ;	Kuvi (Odia)
+kxv_Telu ;	basic ;	Kuvi (Telugu)
```
Comment (srl295) [inline common/properties/coverageLevels.txt:82]: 🎉

File: common/properties/coverageLevels.txt [hunk 2]
```diff
@@ -129,13 +136,15 @@ uk ;	modern ;	Ukrainian
 ur ;	modern ;	Urdu
 uz ;	modern ;	Uzbek
 uz_Cyrl ;	basic ;	Uzbek (Cyrillic)
+vec ;	moderate ;	Venetian
```
Comment (srl295) [inline common/properties/coverageLevels.txt:139]: 🎉

File: common/properties/coverageLevels.txt [hunk 3]
```diff
@@ -129,13 +136,15 @@ uk ;	modern ;	Ukrainian
...
 vi ;	modern ;	Vietnamese
 wo ;	basic ;	Wolof
 xh ;	moderate ;	Xhosa
+xnr ;	basic ;	Kangri
```
Comment (srl295) [inline common/properties/coverageLevels.txt:143]: 🎉

File: common/properties/coverageLevels.txt [hunk 4]
```diff
@@ -18,21 +18,23 @@ be ;	modern ;	Belarusian
...
 br ;	moderate ;	Breton
 brx ;	basic ;	Bodo
 bs ;	modern ;	Bosnian
 bs_Cyrl ;	basic ;	Bosnian (Cyrillic)
 ca ;	modern ;	Catalan
 ceb ;	moderate ;	Cebuano
-chr ;	moderate ;	Cherokee
+chr ;	modern ;	Cherokee
 cs ;	modern ;	Czech
+csw ;	basic ;	Swampy Cree
```
Comment (srl295) [inline common/properties/coverageLevels.txt:31]: 🎉

File: common/properties/coverageLevels.txt [hunk 5]
```diff
@@ -18,21 +18,23 @@ be ;	modern ;	Belarusian
 bg ;	modern ;	Bulgarian
 bgc ;	basic ;	Haryanvi
 bho ;	basic ;	Bhojpuri
+blo ;	moderate ;	Anii
```
Comment (srl295) [inline common/properties/coverageLevels.txt:21]: 🎉

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
@macchiati  please review & merge  https://github.com/macchiati/cldr/pull/82 then this will pass

**Comment by srl295** [comment]
@macchiati please see my sub-pr linked above, it now passes tests

**Comment by macchiati** [comment]
Ah, now I get it. We have a test to make sure that any locales ≥ Basic are in coverage.

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
needed a rebase merge, but you can squash it now

**Pull request merged**